### DISCUSSION TITLE
Share your SCPP projects here!

### DISCUSSION THREAD

**Discussion by Its-Jakey** [Its-Jakey/SCPP#5] [Show and tell]
What programs have you made with SCPP? Share them here! I've made a wireframe 3D engine (it's running on a desktop emulator in the photo, it gets about 2FPS on scratch
![SCPP3D](https://user-images.githubusercontent.com/102316437/178120240-25f34cde-c043-4d56-978c-bb63cd1a0bb9.png)

**Comment by MinekPo1** [comment]
Not with SCPP, but I made a program to analyze jumps in SLVM code! See it [here](https://gist.github.com/MinekPo1/99ae6322787c73699645e952d35be826).

[This](https://gist.github.com/MinekPo1/86fff737618633f7d2b4cfb8bc221f66) is an example tree (for HelloWorldConsole.txt)

Made it because the default compiler includes the entire std by default for some reason and has no dead code elimination and I had a hard time looking through the jumps my self.

**Reply by Its-Jakey** [reply]
fixed the std include problem
